src/transactions/TransactionUtils.cpp

```diff
@@ -230,7 +231,8 @@ addBalance(LedgerTxnHeader const& header, LedgerTxnEntry& entry, int64_t delta)
         {
             return true;
         }
-        if (!isAuthorized(entry))
+
+        if (header.current().ledgerVersion < 10 && !isAuthorized(entry))
```

Comment:
I understand that this "cannot be reached", but as this is super critical code, I'd rather keep it as `!isAuthorizedToMaintainLiabilities(entry)` (defensive)

Reply:
done